.env

```diff
@@ -0,0 +1,17 @@
+##################################################################
+# This file is used by `docker-compose` and sets default values.
+# You can override any of these in your shell, for example by
+# adding export statements to your .bashrc:
+#    export DDS_CLI_ENV="production"
+##################################################################
+
```

Comment:
Can you also add `FLASK_ENV="development"`, since this file is intended for development

Reply:
This is exposed in the `docker-compose.yml` file. But yeah I agree that it would be better to put it here and set a default.